docs/text.coffee.md

```diff
@@ -83,12 +83,15 @@ below.
 * `columnGap` - the amount of space between each column (1/4 inch by default)
 * `indent` - the amount in PDF points (72 per inch) to indent each paragraph of text
 * `paragraphGap` - the amount of space between each paragraph of text
+* `marginBefore` - the amount of space before the text
```

Comment:
Maybe call it "marginTop" and "marginBottom"?

It had me confused with "marginLeft" and "marginRight".